dashboard/test/controllers/api_controller_test.rb

```diff
@@ -1193,9 +1194,10 @@ def make_progress_in_section(script)
       {method: "get", path: "/api/student_progress/2/15"}
     )
 
-    assert_recognizes(
-      {controller: "api", action: "whatevvv"},
-      {method: "get", path: "/api/whatevvv"}
-    )
+# Wildcard routes deprecated in Rails 5
+#    assert_recognizes(
+#      {controller: "api", action: "whatevvv"},
+#      {method: "get", path: "/api/whatevvv"}
+#    )
```

Comment:
Is there any reason to leave this in? Maybe add a TODO to remove the comment one we upgrade to Rails 5?